### ISSUE TITLE
game.get_results() like on sgf file

### ISSUE THREAD

**Issue by sugizo** [atw1020/sente#29]
```
print('Result :', game.get_results() )
```
**result**
```
Result : <sente.result "W+R">
```
**expected result like on sgf file, more human readable**
```
Result : W+R
```

**question**
how to achieve it in sente ?

thanks and best regards,

**Comment by atw1020** [comment]
Casting it to a string might do the trick

```python
>>> print(str(game.get_results()))
```

**Comment by sugizo** [comment]
`print(str(game.get_results()))`

**result**
`<sente.result "W+R">`

**expected result**
W+R

without <sente.result "whatever game result">

**Comment by atw1020** [comment]
I'll make str() work when I fix your other issues, thank you for bringing this to my attention!

**Comment by atw1020** [comment]
Fixed in sente v 0.4.2
